src/rqt_console/console_widget.py

```diff
@@ -849,7 +853,8 @@ def restore_settings(self, pluggin_settings, instance_settings):
             if exclude_filters is not None:
                 for index, item in enumerate(exclude_filters):
                     self._add_exclude_filter(item)
-                    filter_settings = instance_settings.get_settings('exclude_filter_' + str(index))
+                    index_name = 'exclude_filter_' + str(index)
```

Comment:
Might I recommend a format string for readability. I thought there was a space there for a few seconds.
```suggestion
                    index_name = f'exclude_filter_{str(index)}'
```